### PR TITLE
add launch file for video recorder

### PR DESCRIPTION
**Pull request by PetervDooren**
Created a launch file for the video recorder. It adds a timestamp to the end of filenames to make sure they are unique and  dont overwrite one another.

### CODE REVIEW

File: launch/tools/video_recorder.launch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<launch>
+    <!-- Gets current time at launch execution as a formatted string -->
+    <arg name="filename" default="video"/>
+    <arg name="time_now" default="$(eval eval ('_' + '_import_' + '_(\'datetime\')').datetime.now().strftime('%Y-%m-%d-%H-%M-%S'))"/>
+    <arg name="folder" default="/home/amigo/Videos/Challenges"/>
```
Comment (MatthijsBurgh) [inline launch/tools/video_recorder.launch]: Use env to get HOME

File: launch/tools/video_recorder.launch [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <arg name="full_filename" value="$(arg folder)/$(arg filename)_$(arg time_now).avi" />
+
+    <arg name="image_topic"/>
+
+    <node name="video_recorder" pkg="test_tools" type="record_video_from_camera" args="--topic $(arg image_topic) --file $(arg full_filename)"/>
+
+</launch>
```
Comment (MatthijsBurgh) [inline launch/tools/video_recorder.launch]: Missing line at EOF

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
The filename logic should be in the python code IMHO

**Comment by PetervDooren** [comment]
I put the logic here so that the python code does not dictate the filename. That filename should be decided at a layer above the python code. In this case that was the launch file.

**Comment by MatthijsBurgh** [comment]
> I put the logic here so that the python code does not dictate the filename. That filename should be decided at a layer above the python code. In this case that was the launch file.

I think the naming logic of `rosbag` (https://github.com/ros/ros_comm/blob/noetic-devel/tools/rosbag/src/rosbag/rosbag_main.py#L91) could be a solution.

**Comment by MatthijsBurgh** [comment]
When someone else approves the current changes. You can merge this. Though the comments in the other PR are an much bigger concern.

**Pull request closed without merging**